### ISSUE TITLE
Branch management - future developments

### ISSUE THREAD

**Issue by drupol** [ec-europa/ec_europa#105]
There are 3 versions of ECL.
* 0.x, 0.0.24 is the latest
* 1.x, 1.4.0 is the latest
* 2.x, 2.1.1 is the latest

We also have 2 version of Atomium:
* 7.x-2.x, 2.15 is the latest
* 7.x-3.x, 3.0-alpha2 is the latest

We have 2 versions of PHP:
* PHP 5.6
* PHP 7.x (_not decided yet_)

We have multiple NextEuropa branches
* 2.4.x
* 2.5.x
* 2.6.x

The goal of this issue is to decide which version of ECL we are going to support, with which version of Atomium.

I'm almost sure that NextEuropa 2.6.x will support PHP 7, AFAIK 2.4.x and 2.5.x are still using PHP 5.6.

**Comment by kameleon69** [comment]
Yes Florin Bota confirmed we will use PHP7 on 2.6.X : no 5.6 compatibility - move to 2.6 forces move to PHP7

**Comment by drupol** [comment]
Thanks @kameleon69 !
